test/index.js

```diff
@@ -40,8 +42,14 @@ client.connect().then(() => {
     .then(() => db.collection('books').insert({ title: 'Hello 1', rand: Math.random() }))
     .then(() => console.log('inserted document 1'))
     .then(() => reader.start())
-    .then(() => db.collection('books').insert({ title: 'Hello 2', rand: Math.random() }))
-    .then(() => console.log('inserted document 2'))
+    .then(() => 
+      db.collection('books').insert({ title: 'Hello 2', rand: Math.random() })
+        .then(result => {
+          const query = { _id: result.insertedIds[0] };
+          console.log('inserted document 2')
```

Comment:
Can we get rid of this kind of logs or they are set intentionally there?

Reply:
I'd like to keep it to give the tests a bit more verbosity.

Reply:
Alright, my concern based on that i don't really know what this code does. I mean, `inserted document 2` looks something added on dev stage.

Reply:
Every db modification should be emitted by the oplog reader. The test inserts and updates a few documents and counts to make sure the correct number of documents are emitted by the oplog reader.